src/weather_gen.cpp

```diff
@@ -1,20 +1,21 @@
 #include "weather_gen.h"
 
 #include <algorithm>
-#include <chrono>
 #include <cmath>
 #include <ostream>
 #include <random>
 #include <string>
+#include <utility>
 
 #include "cata_utility.h"
 #include "game_constants.h"
 #include "json.h"
```

Comment:
This one is probably mandatory for Windows-builds, so we can guard it with ifdef.
```suggestion
#include "json.h"
#include "math_defines.h"
```